anthos-bm-gcp-bash/setup_and_install_abm.sh

```diff
@@ -0,0 +1,299 @@
+# between them
+printf "🔄 Setting up VxLAN in the GCE VMs...\n"
+# [START anthos_bm_gcp_bash_add_vxlan]
+i=2 # We start from 10.200.0.2/24
+for vm in "${VMs[@]}"
+do
+    gcloud compute ssh root@"$vm" --zone "${ZONE}" << EOF
+        apt-get -qq update > /dev/null
+        apt-get -qq install -y jq > /dev/null
+        set -x
+        ip link add vxlan0 type vxlan id 42 dev ens4 dstport 0
+        current_ip=\$(ip --json a show dev ens4 | jq '.[0].addr_info[0].local' -r)
+        printf "VM IP address is: \$current_ip"
+        for ip in ${IPs[@]}; do
+            if [ "\$ip" != "\$current_ip" ]; then
+                bridge fdb append to 00:00:00:00:00:00 dst \$ip dev vxlan0
+            fi
+        done
+        ip addr add 10.200.0.$i/24 dev vxlan0
+        ip link set up dev vxlan0
+
+EOF
+    i=$((i+1))
+done
+# [END anthos_bm_gcp_bash_add_vxlan]
+printf "✅ Successfully setup VxLAN in the GCE VMs.\n\n"
+
+# install the necessary tools inside the VMs
+printf "🔄 Setting up admin workstation...\n"
+# [START anthos_bm_gcp_bash_init_vm]
+gcloud compute ssh root@$VM_WS --zone "${ZONE}" << EOF
+set -x
+
+export PROJECT_ID=\$(gcloud config get-value project)
+
+gcloud iam service-accounts keys create bm-gcr.json \
+  --iam-account=baremetal-gcr@\${PROJECT_ID}.iam.gserviceaccount.com
+
+curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl"
+
+chmod +x kubectl
+mv kubectl /usr/local/sbin/
+mkdir baremetal && cd baremetal
+gsutil cp gs://anthos-baremetal-release/bmctl/1.13.0/linux-amd64/bmctl .
+chmod a+x bmctl
+mv bmctl /usr/local/sbin/
+
+cd ~
+printf "Installing docker"
+curl -fsSL https://get.docker.com -o get-docker.sh
+sh get-docker.sh
+EOF
+# [END anthos_bm_gcp_bash_init_vm]
+printf "✅ Successfully set up admin workstation.\n\n"
+
+# generate SSH key-pair in the admin workstation VM and copy the public-key
+# to all the other (control-plane and worker) VMs
+printf "🔄 Setting up SSH access from admin workstation to cluster node VMs...\n"
+# [START anthos_bm_gcp_bash_add_ssh_keys]
+gcloud compute ssh root@$VM_WS --zone "${ZONE}" << EOF
+set -x
+ssh-keygen -t rsa -N "" -f /root/.ssh/id_rsa
+sed 's/ssh-rsa/root:ssh-rsa/' ~/.ssh/id_rsa.pub > ssh-metadata
+for vm in ${VMs[@]}
+do
+    gcloud compute instances add-metadata \$vm --zone ${ZONE} --metadata-from-file ssh-keys=ssh-metadata
+done
+EOF
+# [END anthos_bm_gcp_bash_add_ssh_keys]
+printf "✅ Successfully set up SSH access from admin workstation to cluster node VMs.\n\n"
+
+# initiate Anthos on bare metal installation from the admin workstation
+printf "🔄 Installing Anthos on bare metal...\n"
+# [START anthos_bm_gcp_bash_install_abm]
+gcloud compute ssh root@$VM_WS --zone "${ZONE}" <<EOF
+set -x
+export PROJECT_ID=$(gcloud config get-value project)
+export clusterid=cluster-1
+bmctl create config -c \$clusterid
+cat > bmctl-workspace/\$clusterid/\$clusterid.yaml << EOB
```

Comment:
Since it is referred to by region tag in the devsite docs - yes it needs to be embedded. Splitting it will also complicate the gcloud command issued in the workstation - because we must first SSH into the VM and then run the command from within.

Also would not want to change how things appear in the doc now!